root/edit/search_macros.tt

```diff
@@ -453,7 +453,10 @@
 
 [% MACRO predicate_url(field, field_contents) WRAPPER wfield predicate="url" %]
   [% operators([ [ '=', l('is') ]
-                 [ '!=', l('is not') ] ], field_contents) %]
+                 [ '!=', l('is not') ]
+                 [ '~', l('contains')]
+                 [ '!~', l('does not contain')]
```

Comment:
I'm okay with supporting regular expressions, but the operator labels don't seem right, then: I'd name them "matches" and "does not match."

Even then, we don't mention regular expressions anywhere, so I wouldn't expect anybody to know how to use these fields. We could perhaps add a `(help)` link next to the field (like we already do for dates) and have a section in /doc/How_to_use_the_Edit_Search that links to https://www.postgresql.org/docs/16/functions-matching.html#FUNCTIONS-POSIX-REGEXP

Alternatively, we can stick with "contains"/"does not contain," just add `%` before/after the user-entered value in `transform_user_input`, and use the `LIKE` operator. Though that won't let you do stuff like "starts with" or "ends with," only literally "contains."